dev_guide/builds/advanced_build_operations.adoc

```diff
@@ -122,3 +122,123 @@ endif::[]
 If the specified `NodeSelector` cannot be matched to a node with those labels,
 the build still stay in the `Pending` state indefinitely.
 ====
+
+[[dev-guide-chaining-builds]]
+== Chaining Builds
+
+For compiled languages (Go, C, C++, Java, etc.) the dependencies necessary for compilation might increase the size of the
+image or introduce vulnerabilities that can be exploited.
+
+To avoid these problems, a
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[Source-to-Image]
+build can be chained with a
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[Docker]
```

Comment:
Yes they can, but this is about a specific solution to the problem of capturing binary output from one build and including it in a runtime image. Maybe we should have a different title?